scss/civicrm/administer/contribute/_contribute.scss

```diff
@@ -2,16 +2,13 @@
 
 &#{civi-page('admin-contribute')},
 &#{civi-page('admin-pcp')} {
-  .crm-container {
-    div#alpha-filter {
-      @include box-shadow($box-shadow);
-      background: $crm-white;
-      border-bottom: 1px solid $crm-grayblue-dark;
-      border-radius: $border-radius-base $border-radius-base 0 0;
-      margin-bottom: 0;
-      margin-top: 10px;
-      padding: 15px 10px 10px;
-    }
+  #alpha-filter {
+    @include box-shadow($box-shadow);
+    background: $crm-white;
+    border-radius: $border-radius-base $border-radius-base 0 0;
+    margin-bottom: 0;
+    margin-top: 10px;
+    padding: 15px 10px 10px;
```

Comment:
Are there no padding vars associated to these values?

I know that the second `10px` must be associated to this:
https://github.com/civicrm/org.civicrm.shoreditch/blob/master/scss/bootstrap/overrides/_variables.scss

Reply:
I just moved that piece of code from one place to another. But yeah, there is also some code duplication. Fixed.